src/Ens/JobeetBundle/Entity/Job.php

```diff
@@ -0,0 +1,493 @@
+    public function setIsPublic($isPublic)
+    {
+        $this->is_public = $isPublic;
+
+        return $this;
+    }
+
+    /**
+     * Get is_public
+     *
+     * @return boolean 
+     */
+    public function getIsPublic()
+    {
+        return $this->is_public;
+    }
+
+    /**
+     * Set is_activated
+     *
+     * @param boolean $isActivated
+     * @return Job
+     */
+    public function setIsActivated($isActivated)
+    {
+        $this->is_activated = $isActivated;
+
+        return $this;
+    }
+
+    /**
+     * Get is_activated
+     *
+     * @return boolean 
+     */
+    public function getIsActivated()
+    {
+        return $this->is_activated;
+    }
+
+    /**
+     * Set email
+     *
+     * @param string $email
+     * @return Job
+     */
+    public function setEmail($email)
+    {
+        $this->email = $email;
+
+        return $this;
+    }
+
+    /**
+     * Get email
+     *
+     * @return string 
+     */
+    public function getEmail()
+    {
+        return $this->email;
+    }
+
+    /**
+     * Set expires_at
+     *
+     * @param \DateTime $expiresAt
+     * @return Job
+     */
+    public function setExpiresAt($expiresAt)
+    {
+        $this->expires_at = $expiresAt;
+
+        return $this;
+    }
+
+    /**
+     * Get expires_at
+     *
+     * @return \DateTime 
+     */
+    public function getExpiresAt()
+    {
+        return $this->expires_at;
+    }
+
+    /**
+     * Set created_at
+     *
+     * @param \DateTime $createdAt
+     * @return Job
+     */
+    public function setCreatedAt($createdAt)
+    {
+        $this->created_at = $createdAt;
+
+        return $this;
+    }
+
+    /**
+     * Get created_at
+     *
+     * @return \DateTime 
+     */
+    public function getCreatedAt()
+    {
+        return $this->created_at;
+    }
+
+    /**
+     * Set updated_at
+     *
+     * @param \DateTime $updatedAt
+     * @return Job
+     */
+    public function setUpdatedAt($updatedAt)
+    {
+        $this->updated_at = $updatedAt;
+
+        return $this;
+    }
+
+    /**
+     * Get updated_at
+     *
+     * @return \DateTime 
+     */
+    public function getUpdatedAt()
+    {
+        return $this->updated_at;
+    }
+
+    /**
+     * Set category
+     *
+     * @param \Ens\JobeetBundle\Entity\Category $category
+     * @return Job
+     */
+    public function setCategory(\Ens\JobeetBundle\Entity\Category $category = null)
+    {
+        $this->category = $category;
+
+        return $this;
+    }
+
+    /**
+     * Get category
+     *
+     * @return \Ens\JobeetBundle\Entity\Category 
+     */
+    public function getCategory()
+    {
+        return $this->category;
+    }
+    /**
+     * @ORM\PrePersist
+     */
+    public function setCreatedAtValue()
+    {
+          if(!$this->getCreatedAt())
+          {
+              $this->created_at = new \DateTime();
+          }
+    }
+
+    /**
+     * @ORM\PreUpdate
+     */
+    public function setUpdatedAtValue()
```

Comment:
@PhotonSapiens 
it's OK to use this manner for pet project but please use `doctrineExtensionsBundle` instead
Try do it yourself, in case of problems - ask me